.github/workflows/ci.yml

```diff
@@ -27,14 +27,56 @@ jobs:
             - run: yarn circular-dependencies
             - run: yarn build
             - run: yarn ts-check
-            - run: yarn storybook-static
             - run: yarn test --ci
             - run: NO_BUILD=true yarn test-ssr --ci
-            - run: yarn test-acceptance --ci
+
+    build-acceptance:
+        runs-on: ubuntu-latest
+        container: telefonica/web-builder:chromium93.0.4577-1.2.4
```

Comment:
you could update to use the latest image: `chromium93.0.4577-node18-1.0.0`

Reply:
I tried, but had to revert because vercel doesn't support node 18 yet